crates/bevy_ecs/src/system/system_param.rs

```diff
@@ -1304,6 +1305,51 @@ impl<'w, 's> SystemParamFetch<'w, 's> for SystemChangeTickState {
     }
 }
 
+#[derive(Debug)]
+pub struct SystemName {
```

Comment:
Should this implement `Deref<Target=str>`?

Reply:
Probably yeah